src/NekoSite.hx

```diff
@@ -67,21 +83,36 @@ class NekoSite extends Controller {
 		var pvr = new PartialViewResult( {}, "page.html" );
 		return nekoApi.getApi( name ) >> function( page:Page ):ViewResult {
 			pvr.setVars( page );
+			pvr.setVar( "contentGroup", 'doc/libs' );
 			pvr.setVar( "currentYear", Date.now().getFullYear() );
 			pvr.setVar( "editLink", 'https://github.com/HaxeFoundation/nekovm.org/blob/master/www/api/$name.xml' );
 			return pvr;
 		};
 	}
 
+	@:route("/specs/$name")
+	public function specs( name:String ) {
+		var pvr = new PartialViewResult({}, "page.html");
+		return nekoApi.getPage( 'specs/$name' ) >> function( page:Page ):ViewResult {
+			pvr.setVars( page );
+			pvr.setVar( "contentGroup", "specs" );
+			pvr.setVar( "currentYear", Date.now().getFullYear() );
+			pvr.setVar( "editLink", 'https://github.com/HaxeFoundation/nekovm.org/blob/master/www/pages/specs/$name.md' );
+			return pvr;
+		};
+	}
+	
 	@:route("/*")
 	public function page( rest:Array<String> ) {
 		var pageName = (rest.length>0) ? rest.join("/") : "index";
-		var pvr = new PartialViewResult({});
+		var pvr = new PartialViewResult({}, pageName == "index" ? "index.html" : null);
 		return nekoApi.getPage( pageName ) >> function( page:Page ):ViewResult {
```

Comment:
> What's >> for? I don't recognise this syntax.

It maps a future to another value. See https://github.com/haxetink/tink_core#operators